data/id/legislature/Julie-VanOrden-1781e088-14e7-44a6-91fb-7727ecdb1f40.yml

```diff
@@ -0,0 +1,29 @@
+id: ocd-person/1781e088-14e7-44a6-91fb-7727ecdb1f40
+name: Julie VanOrden
+given_name: Julie
+family_name: VanOrden
+email: JVanOrden@senate.idaho.gov
+image: https://legislature.idaho.gov/wp-content/uploads/sessioninfo/2023/directory/VanOrden2129.jpg
+party:
+- name: Republican
+roles:
+- start_date: 2022-01-10
+  end_date: 2022-12-01
+  type: upper
```

Comment:
@bfossen-ce this person served in the house district 31b from 12/01/2012 through 12/01/2018